blueprints/maximum-plaid/index.js

```diff
@@ -0,0 +1,24 @@
+/*jshint node:true*/
+module.exports = {
+  description: 'Installation blueprint for maximum plaid',
+  normalizeEntityName: function () {},
+  beforeInstall: function() {
+    return this.addAddonsToProject({
+      packages: [
+        'ember-d3@~0.2.0',
```

Comment:
No problem, but I'm a huge fan of using a separate addon for things like this. Imagine a case where `maximum-plaid`, `ember-d3-scales`, `ember-d3-shape`, etc... all need to import a version of d3 in order to function properly. They'll all need to do the fastboot stuff independently, they all may add different versions with different features, etc... It turns into a mess. Additionally, it tightly couples the "emberification" of a library (i.e., `maximum-plaid`) with the version of the library its self (i.e., `d3 v3.4.5`).

If they all delegate the addition of this dependency to an addon, this problem goes away. I'll get a new version of `ember-d3` made, with d3 v4 and we can use that